examples/Load/Program.fs

```diff
@@ -9,16 +9,16 @@ open Suave.Http.Applicatives
 open Suave.Http.Files
 open Suave.Logging
 
-let logger = Loggers.sane_defaults_for LogLevel.Verbose
+let logger = Loggers.saneDefaultsFor LogLevel.Verbose
 
-let config : SuaveConfig = 
-  { default_config with 
+let config = 
+  { defaultConfig with 
       bindings = [ HttpBinding.mk' HTTP "127.0.0.1" 8082 ]
-      ; buffer_size = 2048
-      ; max_ops = 10000
-      ; logger = logger }
+      bufferSize = 2048
+      maxOps = 10000
```

Comment:
I think signed is ok, it's pretty normal, but either way let's do it in a follow up PR/discussion?